configs/albu_example/mask_rcnn_r50_fpn_1x.py

```diff
@@ -237,7 +237,7 @@
         # dict(type='TensorboardLoggerHook')
     ])
 # yapf:enable
-evaluation = dict(interval=1)
+evaluation = dict(interval=1, metric=['bbox', 'segm'])
```

Comment:
Better to move it the L221, since evaluation config can be related to the dataset.